content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,130 @@
+
+
+```
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "LoadLevel": "levels/mainmenu/mainmenu.spawnable"
+            }
+        }
+    }
+}
+```
+
+
+## Process assets
+
+The Asset Processor or Asset Processor Batch are responsible for processing source assets into product assets, which the Game Launcher consumes at runtime.
+
+1. Process your project's assets by following either one of the following instructions.
+    
+    - Use **CMake** to build the Asset Processor Batch and its dependencies by specifying the `MyProject.Assets` target. After building, this command runs Asset Processor Batch and processes the assets.
+
+        ```cmd
+        cmake --build build/windows_vs2019 --target MyProject.Assets --config profile -- /m
+        ```
+
+    - Run your project's Asset Processor to begin processing the assets.
+
+
+{{< caution >}} Ensure that the Asset Processor or Asset Processor Batch processes all of your assets without failure. {{< /caution >}}
+
+
+## Create a project game release layout
+
+You can create a project game release layout by using a non-monolithic or monolithic build. A monolithic release layout only contains files that are necessary to run the project, such as the Game Launcher, bundled content, and a minimal set of dependent `.dll` files, if any. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.  Non-monolithic releases are typically much larger than monolithic releases. On some platforms that don't allow loading shared objects, you must use a monolithic release.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory that will contain your release layout.
+
+- Bundles your game content into an `engine.pak` file by packaging all of the product assets in your `Cache/pc` directory.
+
+- Builds the Game Launcher and other supporting files.
+
+{{< todo >}}
+The Game Launcher may fail to load your assets if the `engine.pak` file is over 4 GB. This issue will be resolved when development on the Asset Bundler is stable.
+{{< /todo >}}
+
+### Monolithic project
+
+1. Set up a new CMake build directory, `windows_mono`, for your monolithic build in your project directory. We recommend keeping this build directory separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
+
+    ```cmd
+    cmake -B build/windows_mono -S. -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_MONOLITHIC_GAME=1
```

Comment:
Is this needs to run from source engine directory ot install layout directory? or project directory?